cmake/Modules/FindRTIConnextDDS.cmake

```diff
@@ -365,20 +378,23 @@
 #
 # Supported platforms
 # ^^^^^^^^^^^^^^^^^^^
-# It is compatible with the following platforms listed in the
+# Oficially, this FindPackage supports the following platforms listed in the
 # RTI Connext DDS Core Libraries Platform Notes:
-# - Linux platforms: i86, x64 and ARMv8
-# - Darwin platforms: OS X 10.11-10.13
-# - Windows platforms: i86 and x64
-# - QNX platforms: x64 and ARMv8
-# - Raspbian Wheezy 7.0 (3.x kernel) on ARMv6 (armv6vfphLinux3.xgcc4.7.2)
-# - Android 5.0 and 5.1 (armv7aAndroid5.0gcc4.9ndkr10e)
+#
+# - Linux platforms: x64
+# - Darwin platforms: macOS 10.13-10.15
+# - Windows platforms: x64
+# - QNX platforms (7.0 and 7.1): x64 and ARMv8
+#
+# Other platforms and architectures might work, but they are not oficially
+# supported.
 #
 # Logging in versions lower than CMake 3.15
 # ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
-# For versions lower than CMake 3.15 ``CMAKE_MINOR_VERSION`` variable should be
-# use to define lower logging levels than ``STATUS`` mode. All this modes will
-# show messages of current level and higher. The following modes are available:
+# For versions lower than CMake 3.15, ``CONNEXTDDS_LOG_LEVEL`` variable should
+# be used to define lower logging levels than ``STATUS`` mode. All this modes
+# will show messages of current level and higher. The following modes are
+# available:
```

Comment:
`be used to define logging levels lower than ``STATUS`` mode. All these modes`

Reply:
I don't know why GitHub does not mark this as outdated, I changed the line: https://github.com/rticommunity/rticonnextdds-cmake-utils/pull/52/files#diff-70ac52d42c16dd8a28492615a0aea64bcd83c76eb5107b28c74384cf9d540cbbR395